src/pages/login-page/Main.jsx

```diff
@@ -0,0 +1,44 @@
+export default function Main() {
+  return (
+    <main className="ha-login-gtr">
+      <div className="ha-login-window">
+        <div className="h-login-container">
+          <h2>Login</h2>
+          <form action="inputs">
+            <div className="h-input-text">
+              <label>Email address</label>
+              <input type="text" placeholder="xyz@email.com" />
```

Comment:
Here instead of using type **text**, you can use **email** so it help in validation

Reply:
it will be done while doing validation